AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/UpdateBookingStatusTests.cs

```diff
@@ -473,7 +523,79 @@ public void UpdateBookingStatus_throws_when_bookings_api_exception()
             _mocker.Mock<IBookingsApiClient>().Setup(x => x.UpdateBookingStatusAsync(hearingId, request))
                 .ThrowsAsync(new BookingsApiException("", StatusCodes.Status500InternalServerError, "", null, null));
 
-            Assert.ThrowsAsync<BookingsApiException>(() => _controller.UpdateBookingStatus(hearingId, request));
+            var response = await _controller.UpdateBookingStatus(hearingId, request);
+
+            var result = (BadRequestObjectResult)response;
+            result.StatusCode.Should().Be(StatusCodes.Status400BadRequest);
+            
+        }
+
+        [Test]
+        public async Task UpdateBookingStatus_with_cancellation_returns_bad_request_when_throws_bookings_api_exception_notfound()
```

Comment:
rename to returns not found instead of bad request

Reply:
fixed title of test case to include not found